README.md

```diff
@@ -104,6 +104,10 @@ For projects that need extra control over how versions are incremented `release-
 ```
 Will mean that any breaking change is treated as a minor which is useful in case the project is in the pre 1.0 stage. Please use responsibly
 
+## Custom github api url for ghe
+you can set GITHUB_API_URL to a custom url e.g.
```

Comment:
This docs needs more context and maybe an example of a github enterprise url (which if I remember correctly is the intention of this feature?)

Reply:
Yes, I'll update

Reply:
@mansona better?